queue-manager/queue-manager.go

```diff
@@ -0,0 +1,164 @@
+package queuemanager
```

Comment:
This abstraction is for running operations.  Clearing queue is one such operation. We should name this OperationManager?

Reply:
done